draft-birkholz-rats-corim.md

```diff
@@ -789,10 +793,629 @@ Not included in the full CDDL definition are CDDL dependencies to CoSWID. The fo
+| 128-255  | Specification Required
+{: #tbl-iana-comid-entity-map-items-reg-procedures title="CoMID Entity Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoMID Entity Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.entity-name | RFC-AAAA
+| 1 | comid.reg-id | RFC-AAAA
+| 2 | comid.role | RFC-AAAA
+| 3-255 | Unassigned
+{: #tbl-iana-comid-entity-map-items title="CoMID Entity Map Items Initial Registrations"}
+
+{: #iana-comid-linked-tag-map-items}
+## CoMID Linked-Tag-Map Items Registry
+
+This document defines a new registry titled "CoMID Linked-Tag Map".
+The registry uses integer values as index values for items in 'comid-linked-tag-map' CBOR maps.
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-comid-linked-tag-map-items-reg-procedures title="CoMID Linked Tag Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoMID Linked-Tag Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.linked-tag-id | RFC-AAAA
+| 1 | comid.tag-rel | RFC-AAAA
+| 2-255 | Unassigned
+{: #tbl-iana-comid-linked-tag-map-items title="CoMID Linked-Tag Map Items Initial Registrations"}
+
+{: #iana-comid-triples-map-items}
+## CoMID Triples-Map Items Registry
+
+This document defines a new registry titled "CoMID Triples Map".
+The registry uses integer values as index values for items in 'comid-triples-map' CBOR maps.
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-comid-triples-map-items-reg-procedures title="CoMID triples Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoMID Triples Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.reference-triples | RFC-AAAA
+| 1 | comid.endorsed-triples | RFC-AAAA
+| 2 | comid.identity-triples | RFC-AAAA
+| 3 | comid.attest-key-triples | RFC-AAAA
+| 4-255 | Unassigned
+{: #tbl-iana-comid-triples-map-items title="CoMID Triples Map Items Initial Registrations"}
+
+{: #iana-comid-environment-map-items}
+## CoMID Environment-Map Items Registry
+
+This document defines a new registry titled "CoMID Environment Map".
```

Comment:
comid-environment-map isn't extensible today. Conceptually, 'environments' could morph over time hence it might make sense for it to be extensible. However, until a use case or conventional wisdom can be described to motivate extensibility it should remain non-extensible. The various maps / types within are extensible hence, there may be no need for extensibility at this map.

Reply:
To reinforce Ned's point on "we don't need extensibility here", it's been already pointed out a few times in our discussions one may argue we already have "too much" here in terms of categorisation because instance may be seen as a special case of group...

> The various maps / types within are extensible hence, there may be no need for extensibility at this map.

emphatic +1